UnityWeld/Binding/Internal/AdapterResolver.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using UnityWeld.Ioc;
+
+namespace UnityWeld.Binding.Internal
+{
+    /// <summary>
+    /// Helper class for creating adapters
```

Comment:
Maybe also explain that it attempts to find the weld container via the attribute and use that but falls back to just using the DefaultWeldContainer if it can't.